core-web/libs/sdk/analytics/src/lib/react/hook/useContentAnalytics.ts

```diff
@@ -0,0 +1,32 @@
+import { useContext } from 'react';
+
+import { DotContentAnalyticsCustomHook } from '../../dotAnalytics/shared/dot-content-analytics.model';
+import { isInsideEditor } from '../../dotAnalytics/shared/dot-content-analytics.utils';
+import DotContentAnalyticsContext from '../contexts/DotContentAnalyticsContext';
+
+/**
+ * Custom hook that handles analytics page view tracking.
+ *
+ * @returns {DotContentAnalyticsCustomHook} - The analytics instance used to track page views
+ *
+ */
+export const useContentAnalytics = (): DotContentAnalyticsCustomHook => {
```

Comment:
I don't see this in the example or the lib, are we using it?

Reply:
Used in the example. I delete it.